modules/storages/app/controllers/storages/admin/storages_controller.rb

```diff
@@ -162,6 +165,20 @@ def update
     end
   end
 
+  def change_health_email_notifications
```

Comment:
Why invent new name here? Let it be `def change_health_notifications_enabled`? Then it is not misleading about what it does.

Reply:
Yeah, I went through a few iterations with the name and forgot to update. `change_health_notifications` sounds more fitting to me. Your call 👍🏾